cpu/samd21/cpu.c

```diff
@@ -31,6 +31,20 @@
 #define GEN2_ULP32K         1
 #endif
 
+#ifndef GEN3_ULP32K
+#define GEN3_ULP32K         1
+#endif
+
+#ifndef XOSC32_STARTUP_TIME
+/**
+ * @brief   XOSC32 start up time
+ *
+ * @note    Override this value in your boards periph_conf.h file
+ *          if a different start up time is to be used.
+ */
+#define XOSC32_STARTUP_TIME      6
```

Comment:
This does not need to be 6 for RTT, but maybe for DFLL it does? I didn't change it because I didn't understand the implications, but at least made it configurable.